### DISCUSSION TITLE
Is this pluging working ?

### DISCUSSION THREAD

**Discussion by papajustify** [capacitor-community/google-maps#104] [Q&A] [answered]
Hi, searching for googlemap on capacitor i reached this projet, but after various tries, and even after folowing tutorials, i can not achieve to make it work.
when i look at the code, I can see that the web.ts is full of methods throwing error like :
throw new Error('Method not implemented.');

do I miss something ?

**Comment by rgbaman** [comment, marked as answer]
You need to run it in simulator or on a device. It is for iOS and Android and doesn't support web (i.e. `ionic serve` preview).
